packages/discovery-react-components/src/components/StructuredQuery/components/RuleRow/RuleRow.tsx

```diff
@@ -45,16 +53,26 @@ export const RuleRow: FC<RuleRowProps> = ({
   const showRemoveRuleRowButton =
     structuredQuerySelection.groups[groupId].rows.length > 1 || !isTopLevelGroup;
 
+  // TODO Check out missing prop warning on <ComboBox />
```

Comment:
Is this comment about `items` being required while the `Combobox` is loading/disabled?

Reply:
going to dive into this a bit later, but this prop warning was on the `onChange` prop

Reply:
did this get resolved?

Reply:
Not yet, circling back on this now

Reply:
I see that the `onChange` is a param meant for you to feed a callback and make this component fully controlled. I imagine we'll make use of `onChange` and the `selectedItem` params when were ready to make this component controlled and start assembling our queries. I've inserted a empty function function for now as well as some comments to come back and change this in the issue that tackles assembling the query